CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
 
 set(FFMPEG_PKG_CONFIG_SUFFIX "" CACHE STRING "This suffix uses for FFmpeg component names searches by pkg-config")
 set(AV_ENABLE_STATIC On CACHE BOOL "Enable static build (On)")
+set(AV_BUILD_EXAMPLES Off CACHE BOOL "Build example applications (Off)")
```

Comment:
Could you pleae make this action enabled by default?

Reply:
Ohh, I'm sorry, I didn't realize that made it into the PR, I'll fix it.  I only added it because I am using avcpp as a subproject, and the tests make my list of targets pretty big.

Reply:
The most latest commit on my branch also has support for enabling/disabling shared library building.  The method used is actually a little gross and results in the full tree being built twice.

I'm not sure if you are going to want that in there.